### PR TITLE
Update nodejs-http-client dependency to v2.3.0, closes #317

### LINKED ISSUE #317: Update nodejs-http-client dependency to v2.3.0
https://github.com/sendgrid/nodejs-http-client/releases/tag/v2.3.0

### PR DESCRIPTION
**Pull request by aszx87410**
Also update TROUBLESHOOTING.md, adding code to log error message.

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -80,9 +80,13 @@ To read the error message returned by SendGrid's API:
   request.path = '/v3/mail/send'
   request.body = requestBody
   sg.API(request, function (error, response) {
-    console.log(response.statusCode)
-    console.log(response.body)
-    console.log(response.headers)
+
```
Comment (thinkingserious) [inline TROUBLESHOOTING.md]: I would also like to keep the logging of the response in the case of no error for demonstration purposes. Could you please add that back in? Perhaps just add an `else`. Thanks!

Comment (aszx87410) [inline TROUBLESHOOTING.md]: Good idea!
I have already committed the new code, which include the `else` part.
Thanks.

**Comment by thinkingserious** [comment]
Hi @aszx87410,

Please take a moment to fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSelP4uF-wGJebxVYq1DYw5WzHomEUauab3YiqrshyCkRs0fqw/viewform#start=invite) so we can send you some swag :)

**Comment by aszx87410** [comment]
Hi @thinkingserious,
I have already filled out the form. Thanks!

**Pull request merged**